AuroraEditor/Base/Toolbar/UI/ToolbarAppInfo.swift

```diff
@@ -42,7 +49,7 @@ public struct ToolbarAppInfo: View {
 
                     Text("|")
 
-                    Text("Today at 17:45")
+                    Text("Today at "+getTime())
```

Comment:
Since it's a placeholder i approve this but please for the next time use
```suggestion
                    Text("Today at " + getTime())
```
that's slightly better readable.